service/lib/dinstaller/dbus/software/manager.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+# Copyright (c) [2022] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "dbus"
+
+module DInstaller
+  module DBus
+    module Software
+      # D-Bus object to manage software installation
+      class Manager < ::DBus::Object
+        PATH = "/org/opensuse/DInstaller/Software1"
```

Comment:
Shouldn't this be `/org/opensuse/DInstaller/Software/Manager1`? Similar to we did with storage.

Reply:
I do not see the need to have a `manager` path at all. For instance, we have `/org/opensuse/DInstaller/Language1`. I am not against it, though, but we should define what we prefer.

Reply:
I am not sure, but I would try to follow the same approach for the different services. So, for storage service, I would expect something like:

* `/org/opensuse/DInstaller/Storage1`
* `/org/opensuse/DInstaller/Storage/Proposal1`
* `/org/opensuse/DInstaller/Storage/Actions1`

Reply:
You are right. I will get rid of that client.